config/services.yaml

```
@@ -50,3 +50,10 @@ services:
             - '@Redis'
             - prefix: session_
 
+    # MenuBuilder
+    app.menu_builder:
+        class: App\Builder\MenuBuilder
```

```suggestion
    App\Builder\MenuBuilder:
```
Si on a pas besoin de l'alias